golem_messages/message.py

```diff
@@ -1330,6 +1369,8 @@ def init_messages():
             MessageSubtaskResultAccepted,
             MessageSubtaskResultRejected,
             MessageDeltaParts,
+            MessageAckReportComputedTask,
+            MessageRejectReportComputedTask,

```

Comment:
It's just a suggestion, but please give it a thought. Maybe it's worth creating commong abstract class with `subtask_id`?
```
AbstractReportComputedTask (`__slots__` = ("subtask_id",))
*-> MessageAckReportComputedTask (`__slots__` = ())
*-> MessageRejectReportComputedTask (`__slots__` = ("reason`,)